public/models/harvard_aeon/archival_object_mapper.rb

```diff
@@ -9,18 +9,14 @@ def map_extensions(mapped, item, repository, resource, resource_json)
     end
 
     def request_permitted?(item)
-      Rails.logger.error(item['json']['jsonmodel_type'])
       if item['json']['jsonmodel_type'] == "archival_object" and item['json']['instances'] and item['json']['instances'].length >= 1
         item['json']['instances'].each do |my_instance|
           if my_instance['instance_type'] and my_instance['instance_type'] == "text" and my_instance['subcontainer'] and my_instance['subcontainer']['topcontainer'] and my_instance['subcontainter']['topcontainer']['ref']
```

Comment:
This is full of misspelled words. It should be `sub_container` and `top_container`